.github/workflows/wix.yml

```diff
@@ -57,6 +60,7 @@ jobs:
       - name: Install dependencies
         run: |
           choco install --no-progress wget
```

Comment:
somewhat unrelated but I don't think this is required anymore. As long as we use `pwsh.exe`

Reply:
Ok, I can try a run where I remove the choco install command

Reply:
Just ran a test run, looks like the `choco install` is still necessary:
https://github.com/adoptium/installer/actions/runs/8788854487/job/24117265968?pr=851#step:6:21